lib/srv/db/common/engines.go

```diff
@@ -117,6 +117,9 @@ type EngineConfig struct {
 	DataDir string
 	// GetUserProvisioner is automatic database users creation handler.
 	GetUserProvisioner func(AutoUsers) *UserProvisioner
+	// UpdateProxiedDatabase finds the proxied database by name and provide a
```

Comment:
"provide a function to update its status" perhaps could be worded as "calls the given function with that database".

But this is a rather generic function - the called function does not need to necessarily "update" anything. The implementation is really "call a function with the database identified by name". I'm not sure what other operations besides "update" makes sense though, so I'll leave it up to you as to whether this field should have a more generic name than "Update..."

Reply:
I updated the description a little bit. I didn't come up with a better name so i will leave that as it is unless someone has better options.